src/nodes/agent/handlers/NodesHolePunchMessageSend.ts

```diff
@@ -103,6 +103,7 @@ class NodesHolePunchMessageSend extends UnaryHandler<
         await nodeConnectionManager.relaySignalingMessage(input, {
           host: meta.remoteHost,
           port: meta.remotePort,
+          scopes: ['external'],
         });
       }
     });
```

Comment:
Is this affected by #555 @tegefaulkes?

Reply:
It might be, we do send host and port, but I'm not sure the scope is relevant to our use.